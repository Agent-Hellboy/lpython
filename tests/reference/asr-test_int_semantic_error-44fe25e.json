{
    "basename": "asr-test_int_semantic_error-44fe25e",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_int_semantic_error.py",
    "infile_hash": "79ca7d3f440b2538aa0819f910bea5ef24820d245b2179e1bf4cce6d",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_int_semantic_error-44fe25e.stderr",
    "stderr_hash": "7f8cdca49225f21009063f92136024e83b75a634b7252f6d52b6f7d8",
    "returncode": 2
}