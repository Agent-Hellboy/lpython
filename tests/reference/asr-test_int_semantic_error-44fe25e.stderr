semantic error: Unsupported type annotation: int
 --> tests/errors/test_int_semantic_error.py:4:8
  |
4 |     x: int = 1
  |        ^^^ 
